qcelemental/tests/test_models_results.py

```diff
@@ -65,6 +65,69 @@
     }
 } # yapf: disable
 
+@pytest.fixture(scope="function")
+def result_data_fixture():
+    mol = qcel.models.Molecule.from_data("""
+        O 0 0 0
+        H 0 0 2
+        H 0 2 0
+    """)
+
+    return {
+        "molecule": mol,
+        "driver": "energy",
+        "model": {
+            "method": "UFF"
+        },
+        "return_result": 5,
+        "success": True,
+        "properties": {},
+        "provenance": {
+            "creator": "qcel"
+        }
+    }
+
+
+@pytest.fixture(scope="function")
+def wavefunction_data_fixture(result_data_fixture):
+    bas = basis.BasisSet(name="custom_basis",
+                         center_data=center_data,
+                         atom_map=["bs_sto3g_o", "bs_sto3g_h", "bs_sto3g_h"])
+    c_matrix = np.random.rand(bas.nbf, bas.nbf)
+    result_data_fixture["wavefunction"] = {"basis": bas, "scf_orbitals_a": c_matrix, "orbitals_a": "scf_orbitals_a"}
```

Comment:
shouldn't "orbitals_a" fail to validate as an array with a string on rhs?

Reply:
`orbitals_a` is a pointer to the correct field. Non prefixed quantities are the expected return result and point to the actual field to save one space.